src/gantt.ts

```diff
@@ -365,12 +350,43 @@ module powerbi.extensibility.visual {
                 .append("g")
                 .classed(Selectors.Tasks.class, true);
 
+            // create tasks container
+            this.taskGroup = this.chartGroup
+                .append("g")
+                .classed(Selectors.Tasks.class, true);
+
+            // create axis container
+            this.axisGroup = this.ganttSvg
+                .append("g")
+                .classed(Selectors.AxisGroup.class, true);
+            this.axisGroup
+                .append("rect")
+                .attr("width", "100%")
+                .attr("y", "-20")
+                .attr("height", "40px")
+                .attr("fill", "white");
```

Comment:
No, `width` and `height` property work no correctly on Edge, if it use as CSS rules